Automattic/wp-calypso
# Add Ratings component and Reviews link to Theme screen

Related to #https://github.com/Automattic/dotcom-forge/issues/4561

## Proposed Changes

* Extract the `ReviewsSummary` to its own component
* Add the Rating component including the average rating
* Add the "X Reviews" link

|Before | After|
|-------|------|
|      ![CleanShot 2023-12-01 at 18 17 01@2x](https://github.com/Automattic/wp-calypso/assets/3519124/6c088247-bb3f-4445-8272-7c46ab9e4632)        |      ![CleanShot 2023-12-01 at 18 16 48@2x](https://github.com/Automattic/wp-calypso/assets/3519124/abecaff8-5997-472d-aa42-f19b28b22f6d)      |

## Testing Instructions



* With the following flags enabled `marketplace-reviews-show` and `marketplace-add-review`
* Navigate to a Theme details page with reviews, for example `/themes/hevor`
* Make sure the Rating component, Reviews link and Add Review button are displayed

- Navigate to a Theme without reviews, for example `/themes/makoney`
- Make sure you do not see the Rating and Reviews link components

## Pre-merge Checklist
